onnx_tf/opr_checker.py

```diff
@@ -0,0 +1,133 @@
+import argparse
+import warnings
+import logging
+import os
+
+import tensorflow as tf
+from google.protobuf import text_format
+
+from onnx import defs
+from onnx_tf import opset_version
+from onnx_tf.common import get_output_node_names
+from onnx_tf.common.handler_helper import get_all_frontend_handlers
+from onnx_tf.pb_wrapper import TensorflowNode
+
+warnings.filterwarnings('ignore')
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger()
+
+
+def parse_args(args):
+  parser = argparse.ArgumentParser(
+      description=
+      'This is the checker to check whether model graph '\
+      'operators are supported by the frontend handlers'
+  )
+  parser.add_argument(
+      "--infile",
+      "-i",
+      help="Input file path, can be pb, pbtxt, or ckpt file.",
+      required=True)
+
+  return parser.parse_args(args)
+
+
+def check_opr_support(graph_def):
+  logger.info('Checking for unsupported operators...')
+  node_dict = set()
+  unsupported = set()
+  supported = set()
+  for node in graph_def.node:
+    node_dict.add(node.op)
+
+  node_dict.discard('Placeholder')
+  node_dict.discard('Const')
+
+  logger.info('There are %s unique operators in the model file.',
+              str(len(node_dict)))
+  for k in node_dict:
+    if k not in opset_version.frontend_tf_opset_version:
```

Comment:
I don't think should use `opset_version.frontend_tf_opset_version`. It's a generated doc not a code based solution.

Reply:
I believe opset_version.frontend_tf_opset_version is generated using onnx_tf/common/handler_helper.py get_frontend_coverage(), therefore it is from code, not just documentation.

Reply:
hi @chinhuang007 i think using opset_version is clearly not as reliable as using get_frontend/backend_coverage() directly. Maybe we can settle with the middle ground in which we invoke these methods directly instead of relying on opset_version.py, thus making this more towards a code based solution rather than a doc based one.

We can also eliminate the possibility that someone forgot to run gen_opset.py, causing opset_version to be out of sync.

Reply:
In my opinion, opset_version should be always up-to-date. We need integrity between doc and code. It is not acceptable to create or change handlers not re-running the regen code. Having said that, I changed the checker to use get_frontend_coverage(). Please review.

Reply:
> We need integrity between doc and code. It is not acceptable to create or change handlers not re-running the regen code. 

@chinhuang007 i totally agree. Both of you make valid points, i proposed such change to strike a balance between concerns from both of you. Thanks for moving this patch forward.